src/include/btree.h

```diff
@@ -103,9 +103,10 @@ struct __wt_btree {
 #define	WT_ASSERT_COMMIT_TS_KEYS	0x02u
 #define	WT_ASSERT_COMMIT_TS_NEVER	0x04u
 #define	WT_ASSERT_DURABLE_TS_ALWAYS	0x08u
-#define	WT_ASSERT_DURABLE_TS_NEVER	0x10u
-#define	WT_ASSERT_READ_TS_ALWAYS	0x20u
-#define	WT_ASSERT_READ_TS_NEVER		0x40u
+#define	WT_ASSERT_DURABLE_TS_KEYS   0x10u
```

Comment:
Did you add this yourself? The spacing between the name and value isn't correct. It should be a tab not spaces. The way to add a new flag is to add it at the top of the list (right below the `/* AUTOMATIC FLAG VALUE GENERATION START */`) give it the name you want and add its value as `0x00u`. Then the scripts will automatically alphabetize it and recompute the bit fields. But you need to have the tabs aligned to start.